.github/workflows/build.yml

```diff
@@ -30,11 +30,11 @@ jobs:
       - name: Checkout Branch
         uses: actions/checkout@v2
       - name: Build with Gradle
-        run: ./gradlew build -Dopensearch.version=1.3.0-SNAPSHOT
+        run: ./gradlew build -Dopensearch.version=1.3.1-SNAPSHOT
```

Comment:
Wondering if we can remove the `-Dopensearch.version` and let it use the default similar to https://github.com/opensearch-project/job-scheduler/pull/154#discussion_r833546447?
@dblock ?